src/doc/building_visit/Advanced_Usage.rst

```diff
@@ -71,6 +71,22 @@ library is substituted at run time for OpenGL when running the parallel
 engine to enable scalable rendering. If you specify ``--mesagl`` then
 ``--osmesa`` is unnecessary and ignored if specified.
 
+
+Building Server only
+~~~~~~~~~~~~~~~~~~~~
+Much of the time, the pre-built VisIt executables will suffice for desktop systems but you need a special version of VisIt to run on a remote server and process data there.
```

Comment:
This sentence doesn't quite make sense to me. I think you're trying to motivate doing a server only build for client server, but the "Much of the time, the pre-built VisIt executables ..." doesn't match the second half of the sentence.